README.md

```diff
@@ -0,0 +1,27 @@
+<div align="center">
+<h1>Volume Snapshot Mover</h1>
+A datamover for CSI snapshots
+</div>
+
+Design Proposal (in-progress): https://github.com/openshift/oadp-operator/pull/597/files
+
+###Basic CSI snapshot workflow steps are as follows:
+- Install OADP Operator, have a stateful application running and create a backup, please
+follow the steps specified [here](https://github.com/openshift/oadp-operator/blob/master/docs/examples/csi_example.md)
+- [Install](https://volsync.readthedocs.io/en/stable/installation/index.html) VolSync controller
+- We will be using VolSync's restic option, hence configure a [restic secret](https://volsync.readthedocs.io/en/stable/usage/restic/index.html#id2).
```

Comment:
is it worth mentioning we are expecting the secret gets created with the name - `restic-secret` ?

Reply:
`restic-config` you mean ?

Reply:
No, our code expects the secret name as `restic-secret`. Thats why I thought its worth mentioning that.